### PR TITLE
Update explore-firewalls.md

### PR DESCRIPTION
**Pull request by brandon-wright**
@jbmitch 

Corrected OpenVPN support for vSRX (it is not supported); also corrected NGFW for vSRX, as IPS/IDS will be supported later in 2019.

### CODE REVIEW

File: explore-firewalls.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ Scroll to the right to view the rest of the table!
 |**Multi-VLAN Support**|No|Yes|Yes|Yes|No|No|No|
 |**NAT Support**|No|Yes|Yes|Yes|Yes|No|No|
 |**SSL/IPsec VPN Termination**|No|Yes|Yes|Yes|Yes|No|No|
-|**Open VPN Termination**|No|Yes|Yes|No|No|No|No|
+|**Open VPN Termination**|No|No|Yes|No|No|No|No|
 |**HA Option**|N/A|Yes|Yes|Yes|Yes|No|Yes|
 |**Manage from API & Portal**|Yes|Appliance GUI|Appliance GUI|Appliance GUI|Appliance GUI|Yes|Yes|
 |**10Gbps Support**|N/A|Yes|Yes|Yes|No|No|No|
-|**NGFW Add-ons (IPS, AV, WF)**|No|No|No|Yes|Yes|No|No|
+|**NGFW Add-ons (IPS, AV, WF)**|No|Yes(2019)|No|Yes|Yes|No|No|
```
Comment (lilorozco) [inline explore-firewalls.md:41]: Should we say 2019 here or is it implied that the capability exists today? or maybe we should say **New!!**

Comment (jbmitch) [reply explore-firewalls.md:41]: @brandon-wright ^

Comment (brandon-wright) [reply explore-firewalls.md:41]: @ibm-cos-help -- We're just trying to highlight that the product supports these features and we're looking to provide access to them via add-on in 2019. Was trying to keep it simple for the matrix. Any thoughts on how to better convey this?

Comment (lilorozco) [reply explore-firewalls.md:41]: If the product supports it now, then a "Yes" should suffice, without a year. If there are caveats to the support, then let's add a few stars and a footnote. @jbmitch should be able to help with that

**Pull request merged**